### PR TITLE
Blogs page

### PR DESCRIPTION
**Pull request by wevekamola**
PR includes
- Blog Page UI
- Show more Blog functionality
- Newsletter Form

### CODE REVIEW

File: public/wp-content/themes/glific/dist/images/blog-delaut-image.svg [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Satendra-SR) [inline public/wp-content/themes/glific/dist/images/blog-delaut-image.svg]: Please optimize and minify. Also, please self-review once.

File: public/wp-content/themes/glific/dist/images/blog-delaut-image.svg [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+<svg width="871px" height="520px" viewBox="0 0 871 520" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```
Comment (Satendra-SR) [inline public/wp-content/themes/glific/dist/images/blog-delaut-image.svg]: Hardcoded height/width?

File: public/wp-content/themes/glific/functions.php [hunk 3]
```diff
@@ -99,9 +116,45 @@ function get_youtube_embed_url($url){
...
 		$youtube_id = $matches[count($matches) - 1];
 	}
-	return 'https://www.youtube.com/embed/' . $youtube_id. '?rel=0' ;
+	return 'https://www.youtube.com/embed/' . $youtube_id . '?rel=0';
+}
+
+function show_more_blogs()
+{
+	if (!isset($_POST['offset']) || empty($_POST['offset'])) {
+		wp_die();
```
Comment (Satendra-SR) [inline public/wp-content/themes/glific/functions.php:125]: Why we are ending up the request without notifying the end-user?  is it a WP practice?

**Review by Satendra-SR — CHANGES_REQUESTED** [review]
@wevekamola minor changes required, please have a look

File: public/wp-content/themes/glific/dist/images/blog-delaut-image.svg (hunk 1 above)
Comment (wevekamola) [reply public/wp-content/themes/glific/dist/images/blog-delaut-image.svg]: 👍

File: public/wp-content/themes/glific/dist/images/blog-delaut-image.svg (hunk 2 above)
Comment (wevekamola) [reply public/wp-content/themes/glific/dist/images/blog-delaut-image.svg]: 👍

File: public/wp-content/themes/glific/functions.php (hunk 3 above)
Comment (wevekamola) [reply public/wp-content/themes/glific/functions.php:125]: @Satendra-SR this function is called from the ajax request. Once the error is sent out from here and received on the ajax call, the error message is sent out from there to the user. Let me know if you have any more queries on this.

**Review by Satendra-SR — APPROVED** [review]
Looks good

**Pull request merged**